SD-K9/src/app/providers/pathfinder.service.ts

```diff
@@ -17,13 +19,18 @@ export class Pathfinder {
    */
   public async getShortestPath(
     pointA: SVGCoordinate,
-    pointB: SVGCoordinate,
-    building: string,
-    floor: number
+    pointB: SVGCoordinate
   ): Promise<SVGCoordinate[]> {
+    console.log(pointA.floor)
+    console.log(pointB.floor)
+    if (pointA.building !== pointB.building || pointA.floor !== pointB.floor) {
+      throw new Error('The start and end points should be in the same floor of the same building.')
```

Comment:
I think it'll mostly be developpers to raise the error since the user will not directly reach this function.
So in this case, it's usually nice to return a question back to the developper.
```suggestion
      throw new Error('Are you sure the start and end points are in the same floor of the same building?')
```